src/features/auth/screens/SignInScreen/components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC, memo, useState } from "react";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+
+interface ForgotPasswordProps {
+  open: boolean;
+  handleClose: () => void;
+  resetPassword: (email: string) => void;
```

Comment:
maybe it is better to have the api logic for resetting password inside this component. This is obviously a temporarily solution because eventually we will move it to redux, but just to make it contained.